supervisor/addons/__init__.py

```diff
@@ -306,22 +306,24 @@ async def update(self, slug: str) -> None:
     async def rebuild(self, slug: str) -> None:
         """Perform a rebuild of local build add-on."""
         if slug not in self.local:
-            _LOGGER.error("Add-on %s is not installed", slug)
-            raise AddonsError()
+            raise AddonsError(f"Add-on {slug} is not installed", _LOGGER.error)
         addon = self.local[slug]
 
         if addon.is_detached:
-            _LOGGER.error("Add-on %s is not available inside store", slug)
-            raise AddonsError()
+            raise AddonsError(
+                f"Add-on {slug} is not available inside store", _LOGGER.error
+            )
         store = self.store[slug]
 
         # Check if a rebuild is possible now
         if addon.version != store.version:
-            _LOGGER.error("Version changed, use Update instead Rebuild")
-            raise AddonsError()
+            raise AddonsError(
+                f"Version changed, use Update instead Rebuild", _LOGGER.error
```

Comment:
```suggestion
                "Version changed, use Update instead Rebuild", _LOGGER.error
```

Only prefix the string with `f` if it contains variables.